analytic_resource_plan/wizard/resource_plan_line_change_state_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="resource_plan_line_change_state_form" model="ir.ui.view">
+        <field name="name">resource.plan.line.change.state.form</field>
+        <field name="model">resource.plan.line.change.state</field>
+        <field name="arch" type="xml">
+            <form string="Change status of Resource Plan Lines">
+                <group>
+                    <field name="state" />
+                </group>
+                <footer>
+                    <button
+                        string="Execute"
+                        name="change_state_confirm"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    <button string="Cancel" class="oe_link" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record
+        id="action_resource_plan_line_change_state_form"
+        model="ir.actions.act_window"
```

Comment:
This action is not showing when selecting the resource plan lines in a list view and clicking on More > Action Can you check?

Reply:
That's good, but the actions are still not showing to me. Are they showing to you? They should show in the resource plan line list.